addons/point_of_sale/tests/test_frontend.py

```diff
@@ -616,6 +616,10 @@ def test_02_pos_with_invoiced(self):
         self.assertEqual(last_order.lines[0].price_subtotal, 30.0)
         self.assertEqual(last_order.lines[0].price_subtotal_incl, 30.0)
 
+        # Check if session name contains config name as prefix
+        last_session = self.env['pos.session'].search([], limit=1, order="id desc")
+        self.assertEqual(self.main_pos_config.name in last_session.name, True)
```

Comment:
```suggestion
        self.assertEqual(self.main_pos_config.name in last_session.name, True)
```

you can use `last_order`'s session